pixart.js

```diff
@@ -1 +1,22 @@
+var colorInput = document.querySelector("#color-field");
+var brush = document.querySelector(".brush");
+var button = document.querySelector("#set-color");
 
+button.addEventListener("click", function(event){
+  event.preventDefault();
+  brush.style.backgroundColor = colorInput.value //why backgroundColor when the css file sets color by background?
+}) //why would this also set up a listener for the enter key
```

Comment:
It shouldn't, an event listener for 'click' will only respond to clicks. If this was a listener for the 'submit' event then both clicks and pressing enter would trigger it.